src/ResultsActivity/ResultsActivity.js

```diff
@@ -380,9 +481,14 @@ ResultsActivity.propTypes = {
   showModal: PropTypes.func.isRequired,
   activityName: PropTypes.string.isRequired,
   updateAttendanceEnabled: PropTypes.bool.isRequired,
+  userRoles: PropTypes.arrayOf(PropTypes.string.isRequired).isRequired,
 }
 
+const mapStateToProps = state => ({
+  userRoles: state.app.user.roles,
+})
+
 const ResultsActivityWithRouter = withRouter(ResultsActivity)
 
 export { ResultsActivity }
-export default ResultsActivityWithRouter
+export default connect(mapStateToProps)(ResultsActivityWithRouter)
```

Comment:
Pattern tends to be to get stuff from the store in the Container above and pass it down as a prop.